### PR TITLE
Beta version of the flutter package

### PR DESCRIPTION
**Pull request by rurickdev**
This PR adds

- Script to generate the icon files
- An example app
- Readme with instructions to use
- Update process instructions
- Screenshots of the example app using the icons
- Icons preview inside IDE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,52 @@ Phosphor is a flexible icon family for interfaces, diagrams, presentations — w
...
 ## Usage
 
-> **TDB**
+### Normal Icons
+
+Just add a `Icon()` widget passing any Phosphor Icon value
+
+```dart
+Icon(
+  PhosphorIcons.pencilLine,
```
Comment (rektdeckard) [inline README.md:29]: Can you add examples for styling here?

File: tools/generate_app.dart [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    '\n  static Map<String, PhosphorIconData> get fillIcons => {',
+    '};'
+  ];
+
+  final iconsGetterContent =
+      '\n  static List<PhosphorIconData> get icons => allIconsAsMap.values.toList();';
+  final namesGetterContent =
+      '\n  static List<String> get names => allIconsAsMap.keys.toList();';
+
+  icons.forEach((icon) {
```
Comment (rektdeckard) [inline tools/generate_app.dart:61]: Would be nice to add fail-fast behavior here -- if all 5 weights are not present, codegen should fail catastrophically.

File: tools/generate_icons.dart [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  final fileLines = [
+    """// Auto Generated File
+import 'package:phosphor_flutter/src/phosphor_icon_data.dart';
+
+abstract class PhosphorIcons {""",
+  ];
+  final constantsLines = <String>[];
+
+  icons.forEach((icon) {
```
Comment (rektdeckard) [inline tools/generate_icons.dart:21]: Ditto here, let's fail if all 5 weights are not present on each icon.

**Review by rektdeckard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rurickdev) [reply README.md]: fixed in lasts comments

File: tools/generate_app.dart (hunk 2 above)
Comment (rurickdev) [reply tools/generate_app.dart:64]: thanks to this I could see a bug in the generator. I fixed it and now works ans checks if an icon don't have the five styles

File: tools/generate_icons.dart (hunk 3 above)
Comment (rurickdev) [reply tools/generate_icons.dart:24]: ditto, check for missing styles and used the new regex

File: generate_files.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+echo "Generation started using Phosphor.json file"
+
+echo "Generating package phosphor_icons.dart file"
+dart ./tools/generate_icons.dart ./Phosphor.json
```
Comment (rektdeckard) [inline generate_files.sh:8]: Update to reflect renamed directory?

Comment (rurickdev) [reply generate_files.sh]: Fixed!

**Pull request merged**